helpers.go

```diff
@@ -339,9 +339,6 @@ func extractTag(dataType reflect.Type) (map[string]tagInfo, []string, error) {
 // Normally, it's used to set the ID field of the target struct
 func setIntValue(target, source reflect.Value) {
 	if target.Kind() == reflect.Ptr {
-		if target.IsNil() {
-			target.Set(reflect.New(target.Type().Elem()))
-		}
```

Comment:
this condition is unnecessary because all the field will be set with non-zero value by `setAssValue`